include/gatekeeper_net.h

```diff
@@ -348,6 +359,24 @@ struct net_config {
 	 * The fields below are for internal use.
 	 * Configuration files should not refer to them.
 	 */
+
+	/*
+	 * Used to calculate the optimal values of gatekeeper_num_mbuf
+	 * and gatekeeper_per_core_cache_size.
+	 */
+	uint8_t              gatekeeper_num_ports;
+	uint16_t             gatekeeper_num_queues;
+	uint16_t             gatekeeper_max_pkt_burst;
+	uint16_t             gatekeeper_num_desc;
```

Comment:
Adding comments on the meaning of these fields would be useful. My suggestions (feel free to improve them):
* `gatekeeper_num_ports`: total number of network ports on all interfaces. Notice that each port corresponds to a physical cable, but the association between ports and physical network interfaces is not obvious.
* `gatekeeper_num_queues`: the maximum number of RX or TX queues on any interface.
* `gatekeeper_max_pkt_burst`: the maximum burst size of any functional block on any interface.
* `gatekeeper_num_desc`: max(INTERFACE.gatekeeper_num_rx_desc + INTERFACE..gatekeeper_num_tx_desc) over all interfaces.